internal/rukpak/source/containers_image.go

```diff
@@ -250,6 +302,11 @@ func (i *ContainersImageRegistry) unpackImage(ctx context.Context, unpackPath st
 	if err != nil {
 		return fmt.Errorf("error creating image source: %w", err)
 	}
+	defer func() {
+		if err := layoutSrc.Close(); err != nil {
+			panic(err)
```

Comment:
just a question - imho panicking here seems a little excessive, but I see it in multiple places, so I think it's getting caught

Reply:
Just following convention here :) Not sure it's the right way to handle it either but I just noticed the source close was missing so I wanted to catch that while I was in here.

Reply:
maybe let's proceed with the convention for this PR, but create a discussion in upstream slack? I'm also a bit :thinking: about these panics. Maybe we should level set on a tenet for when to use it and why?